torch/csrc/jit/passes/inliner.cpp

```diff
@@ -50,8 +51,10 @@ void inlineCalls(Block* block) {
 
 void Inline(Graph& graph) {
   GRAPH_DUMP("Before Inlining: ", &graph);
+  std::cout << "before inline: " << graph.toString() << std::endl;
   inlineCalls(graph.block());
   GRAPH_DUMP("After Inlining: ", &graph);
+  std::cout << "after inline: " << graph.toString() << std::endl;
```

Comment:
Nit: you don't need to insert these prints and could use the existing `GRAPH_DUMP` statements. To turn them on, set the environment variable `PYTORCH_JIT_LOG_LEVEL=inliner`.

More info about the JIT logging could be found here: https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/OVERVIEW.md#jit-logging

Reply:
Thanks! I was trying to find out how to let `GRAPH_DUMP` work and probably miss this setting option..